src/Core/Settings.cpp

```diff
@@ -3148,6 +3148,9 @@ Allow execute multiIf function columnar
 )", 0) \
     DECLARE(Bool, formatdatetime_f_prints_single_zero, false, R"(
 Formatter '%f' in function 'formatDateTime' prints a single zero instead of six zeros if the formatted value has no fractional seconds.
+)", 0) \
```

Comment:
Let's also update the "Notes" below the table in the docs of function "formatDateTime": https://clickhouse.com/docs/en/sql-reference/functions/date-time-functions#formatdatetime

Reply:
Thanks. Done.